packages/web-components/IMPLEMENTATION_NOTES.md

```diff
@@ -291,6 +300,36 @@ The `import`s of `@carbon/ibmdotcom-services-store` code in `@carbon/ibmdotcom-w
 import { loadLanguage, setLanguage } from '../../internal/vendor/@carbon/ibmdotcom-services-store/actions/localeAPI';
 ```
 
+## Rollup bundle for Dotcom Shell
+
+For quick setup of project using minimum feature set (Dotcom Shell) of `@carbon/ibmdotcom-web-components`, a [Rollup bundle](https://unpkg.com/browse/@carbon/ibmdotcom-web-components@1.0.0/dist/ibmdotcom-web-components-dotcom-shell.min.js) is provided.
+
+The Rollup bundle builds the [entry point (`ibmdotcom-web-components-dotcom-shell.ts`)](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/v1.15.0/packages/web-components/src/globals/ibmdotcom-web-components-dotcom-shell.ts). Whenever the Dotcom Shell contains a new components, etc. the entry point file must be updated.
```

Comment:
@asudoh for all of the links, can you point to master instead of the version number (v1.15.0)? It may be hard to keep up with updating the links with every version release.

Reply:
As discussed offline, we are good with targeting the specific release version at this time. Thanks @asudoh !